<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profile</class>
 <widget class="QDialog" name="profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(53, 158, 158);</string>
  </property>
  <widget class="QLabel" name="university_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>university:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="university_line_edit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(193, 193, 193);
color: rgb(36, 36, 36);</string>
   </property>
  </widget>
  <widget class="QLabel" name="company_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>company:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="company_line_edit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(193, 193, 193);
color: rgb(36, 36, 36);</string>
   </property>
  </widget>
  <widget class="QLabel" name="job_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>55</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>job:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="job_line_edit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(193, 193, 193);
color: rgb(36, 36, 36);</string>
   </property>
  </widget>
  <widget class="QLabel" name="birthdate_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Birth date:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="day_line_edit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(193, 193, 193);
color: rgb(36, 36, 36);</string>
   </property>
   <property name="inputMask">
    <string>D9</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="month_line_edit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(193, 193, 193);
color: rgb(36, 36, 36);</string>
   </property>
   <property name="inputMask">
    <string>D9</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="year_line_edit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(193, 193, 193);
color: rgb(36, 36, 36);</string>
   </property>
   <property name="inputMask">
    <string>D999</string>
   </property>
  </widget>
  <widget class="QLabel" name="day_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>40</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>day</string>
   </property>
  </widget>
  <widget class="QLabel" name="month_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>month</string>
   </property>
  </widget>
  <widget class="QLabel" name="year_label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>50</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>year</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_button">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(28, 83, 83);</string>
   </property>
   <property name="text">
    <string>Update profile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_button">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(28, 83, 83);</string>
   </property>
   <property name="text">
    <string>log out</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
